template/virTemplate.md

```diff
@@ -1,40 +1,140 @@
+
+## Characterizing Clostridiae-associata centralis and its relation to Clostridium beijerinckii
+
+## Finding open reading frames (ORFs) and predicting protein structure
+
+I ran NCBI ORFfinder on Clostridiae-associata centralis sequencing data, and identified 4 ORFs (see Fig. 1). I then used AlphaFold3 to predict protein structure. Prediction with all 4 ORFs gave values of ipTM = 0.25 pTM = 0.53, indicating a low-confidence prediction of interaction between the ORFs [EMBL-EBI 2024]. This low-confidence prediction is likely partly due to the fact that ORF2 and ORF4 are not complete CDS. Structure prediction of ORF2 alone showed a confident structure prediction for conserved Oblin-1 structures, but low-confidence prediction of the structure elsewhere in the protein (see Fig. 3). It is thus likely that the sequence encoding the low-confident regions is incomplete, and the structure of these regions may mirror the structure of Oblin-1 when complete.
+
+## Sequence comparison of Clostridiae-associata centralis with experimental sequencing data from BioProject 
+
+To investigate whether Clostridiae-associata centralis is present in the Clostridium beijerinckii genome, I performed a tBLASTn search [NCBI BLAST] with my query sequence as ORF2 from Clostridiae-associata centralis and my chosen search set as the experimental data associated with BioProject PRJNA315559 (ERX2023530). My goal was to see if the putative Oblin-1 ORF in Clostridiae-associata centralis was present in the BioProject sequencing data, which would provide insight as to whether functional sequence from Clostridiae-associata centralis could be found in Clostridium beijerinckii. The result returned over 100 alignments with percent identity over 85% and E values under 0.01 (see Fig. 2).
+
+![Fig. 2: Results from tBLASTn with Clostridiae-associata centralis ORF2 query and ERX2023530 chosen search set.](img/alignment.png)
+
 
-{Q2 answer ...}
 
 ### Virus Genome {Q3}
 
-{Insert your Virus Genome Map Figure and figure legend here}
+![Fig. 1A: ORFs of Clostridiae-associata centralis as determined by ORFfinder, with nucleotide scale bar. Created with R package gggenes based on GitHub PuravG/Computational-Virus-Discovery](img/genome.png)

```

Comment:
if the "Figure Legend" is in the title, it won't be displayed, The square-bracket text here is the "title" shown if the image does not load. Instead put the text in italics underneath the figure.